### ISSUE TITLE
Combination of vflip and compact rendering results in broken rendering

### ISSUE THREAD

**Issue by whitequark** [drom/wavedrom#381]
This can be seen in the [editor](https://wavedrom.com/editor.html). To reproduce:

```json
{
  "reg": [
  	{"name": "first",  "bits": 3},
  	{"name": "second", "bits": 7},
  	{"name": "third",  "bits": 6},
  	{"name": "",       "bits": 16}  
  ], 
  "config": {
  	"lanes": 2,
        "compact": true,
  	"vflip": true
  }
}
```

![Screenshot_20230221_210328](https://user-images.githubusercontent.com/54771/220457978-be7fa69d-7344-49a4-ae4e-f5df4d2edecc.png)

**Comment by drom** [comment]
@whitequark should be fixed in the v3.2.0

**Comment by whitequark** [comment]
Thanks a lot!
